packages/apex-lsp-testbed/src/cli.ts

```diff
@@ -19,6 +19,163 @@ const {
 const { createClient } = require('./utils/clientFactory');
 const { startInteractiveMode } = require('./utils/interactiveMode');
 
+/**
+ * Run node server tests
+ */
+async function runNodeServerTests(client: any, logger: any): Promise<void> {
+  try {
+    logger.info('Starting Node Apex Language Server tests...');
+
+    // Get server capabilities
+    const capabilities = client.getServerCapabilities();
+    logger.info(
+      `Server capabilities: ${JSON.stringify(capabilities, null, 2)}`,
+    );
+
+    // Run basic document test
+    await testBasicDocument(client, logger);
```

Comment:
Do we always want to run all tests here? or is there no situation in which we'd want to specify any subset?

Reply:
I don't think it needs to be done on this pr for the introduction of the tests themselves but I'm imagining some control over which ones run in the future would be good with an options object or something passed into it which could be set by an orchestrator of sorts. Perhaps a discussion for a later time though.